userspace/libscap/compat/bpf.h

```diff
@@ -14,6 +14,7 @@
 /* Extended instruction set based on top of classic BPF */
 
 /* instruction classes */
+#define BPF_JMP32	0x06	/* jmp mode in word width */
```

Comment:
I guess it's okay to not mention where this file came from. Probably better to keep as is than to modify just to add a not saying where it was copied from.